src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/CertificateTests.cs

```diff
@@ -58,8 +58,7 @@ private static void VerifyCertificates0(byte[] encodedMessage)
             Assert.Equal(0, certs.Count);
         }
 
-        [Fact]
-        [ActiveIssue(3334, PlatformID.AnyUnix)]
+        [ConditionalFact(nameof(SupportsKeyAgreementCerts))]
```

Comment:
This one seems like it's really about supporting originatorInfo.  Perhaps a comment to that effect here would be good?